test/fixtures/gcs-fixture/src/main/java/fixture/gcs/GoogleCloudStorageHttpHandler.java

```diff
@@ -167,13 +167,13 @@ public void handle(final HttpExchange exchange) throws IOException {
                 final StringBuilder batch = new StringBuilder();
                 for (String line : Streams.readAllLines(requestBody.streamInput())) {
                     if (line.length() == 0 || line.startsWith("--") || line.toLowerCase(Locale.ROOT).startsWith("content")) {
-                        batch.append(line).append('\n');
+                        batch.append(line).append("\r\n");
```

Comment:
This was also broken, I found it in my wireshark adventures during test debugging. We obviously need CLRF in a multi-part response.